catalog/app/containers/Auth/saga.js

```diff
@@ -468,6 +486,7 @@ function* handleAuthLost({ forgetTokens, forgetUser, onAuthLost }, { payload: er
 function* handleSignUp({ payload: credentials, meta: { resolve, reject } }) {
   try {
     yield call(signUp, credentials)
+    // TODO: yield call(setBrowseCookie, tokens)
```

Comment:
probably not required here

Reply:
I'm curious - how does sign up work? (I'm assuming, this is only password signup for public stacks.) Shouldn't it call storeTokens, etc. at some point?

Reply:
afair it creates the user and then shows the message to check your email to complete the registration, and then you can sign-in

Reply:
Oh right